gen/kneeboard.py

```diff
@@ -230,28 +230,36 @@ def write(self, path: Path) -> None:
             "#", "Action", "Alt", "Dist", "GSPD", "Time", "Departure"
         ])
 
-        writer.heading("Comm Ladder")
-        comms = []
+        writer.table([
+            ["{}lbs".format(self.flight.bingo_fuel), "{}lbs".format(self.flight.joker_fuel)]
+        ], ['Bingo', 'Joker'])
+
+        # Package Section
+        writer.heading("Comm ladder")
+        packageTable = []
```

Comment:
`comm_table` is more accurate. None of this is actually related to the package.

Reply:
Yeah, my original idea is to add all the comm ladder including the package comms. But for know ill change it.